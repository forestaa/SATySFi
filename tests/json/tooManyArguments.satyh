let-math \f = ${2} in ${\f{hoge}}
