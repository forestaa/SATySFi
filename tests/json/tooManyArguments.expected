{
  "file": "toomanyarguments.satyh",
  "range": {
    "start": { "line": 1, "character": 24 },
    "end": { "line": 1, "character": 32 }
  },
  "message":
    "! [Type Error] too many argument(s);\n    the command has type\n      [] math-cmd."
}
